kal-aalrajhi/overlook-hotel
# Feature/manager

## Description 
Users can now login as a **manager**. The manager view permits many privileges like seeing the day's stats, adding/deleting bookings for customers, and even viewing customer stats. You can finally feel like a real space cowboy and manage your hotel properly!

#### Where to start?
Login using the following
- Username: *manager*
- Password: *overlook2021*

#### How to test?
To run the server used, locally, please clone and [follow the instructions from this repo](#https://github.com/turingschool-examples/overlook-api).

run `npm start`

Login with the manager's credentials and then, at the dashboard, search for a customer by name. Once the customer appears, you can add or delete bookings for them. Note that you can only delete future bookings.

#### Screenshots 
**Manager login**
<img width="637" alt="manager_login" src="https://user-images.githubusercontent.com/97044701/165349994-5dfba789-6b45-4d24-a775-dc873a33da14.png">

**Hotel and user stats display**
<img width="325" alt="stats" src="https://user-images.githubusercontent.com/97044701/165350006-e555af49-21fe-4bda-b5d0-134bc93b56d1.png">

**Delete bookings**
<img width="808" alt="delete" src="https://user-images.githubusercontent.com/97044701/165350022-cfaa6baa-0f24-47b8-9316-fa103fa076e6.png">

**Adding bookings for customers**
<img width="737" alt="book" src="https://user-images.githubusercontent.com/97044701/165350038-b0adefe7-163a-4876-9ce8-ee0fef660f54.png">

#### Type of Change:

- [] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (code is working and without bugs, formatting has been streamlined)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?

What steps have you taken to ensure the code is effective?

#### Checklist:
